cluster/kubemark/gce/config-default.sh

```diff
@@ -24,6 +24,8 @@
 source "${KUBE_ROOT}/cluster/gce/config-common.sh"
 
 CLEANUP_KUBEMARK_IMAGE=${CLEANUP_KUBEMARK_IMAGE:-true}
+TEST_CLUSTER_LOG_LEVEL="${TEST_CLUSTER_LOG_LEVEL:-"--v=4"}"
+KUBE_GCE_INSTANCE_PREFIX="${KUBE_GCE_INSTANCE_PREFIX:-"default"}"
```

Comment:
While the TEST_CLUSTER_LOG_LEVEL totally make sense, I'm not sure about this one.

KUBE_GCE_INSTANCE_PREFIX serves as kind of obligatory parameter that should be set by user. It allows you to set the "name" of the cluster, in a way that if you set a different names for two different start-kubemark.sh invocations, you'll get two different kubemark clusters, while if the name is the same the second call would most likely fail.

If we set the default for it, we should at least document that if you want to set up two kubemark clusters with different "names" you should change this env per each start-kubemark.sh invocation.

Reply:
Thanks, you are right. I've added the document.

PS: Previously, the name was defined by 

> INSTANCE_PREFIX="${INSTANCE_PREFIX:-"default"}"
MASTER_NAME="${INSTANCE_PREFIX}-kubemark-master"